pkg/component/observability/plutono/plutono.go

```diff
@@ -144,21 +145,19 @@ type plutono struct {
 }
 
 func (p *plutono) Deploy(ctx context.Context) error {
-	dashboardConfigMaps, data, err := p.computeResourcesData(ctx)
+	dashboardConfigMap, data, err := p.computeResourcesData(ctx, plutonoAdminUserSecret)
```

Comment:
We see that under `pkg/component/observability/plutono/dashboards/garden/` there are still the 2 dirs - garden and global. We were wondering whether all of the dashboards should be merged in the same dir (`pkg/component/observability/plutono/dashboards/garden/`) because we no longer have the dir structure in plutono. It might be confusing for someone to see these `garden`/`global` separation in the file system in the git repo. WDYT?